src/DateInput/MonthInput.jsx

```diff
@@ -47,8 +47,8 @@ export default class MonthInput extends PureComponent {
         onChange={onChange}
         onFocus={event => select(event.target)}
         onKeyDown={onKeyDown}
-        onKeyUp={event => updateInputWidth(event.target)}
-        placeholder="--"
+        onKeyUp={event => appendInputValue(event, onChange)}
+        placeholder="mm"
```

Comment:
Let's focus on one change at a time. Changing placeholders is discussed in another issue.

Reply:
Agreed and reverted.